go/integTest/shared_commands_test.go

```diff
@@ -172,6 +173,76 @@ func (suite *GlideTestSuite) TestSetWithOptions_ReturnOldValue_nonExistentKey()
 	})
 }
 
+func (suite *GlideTestSuite) TestMSetAndMGet_existingAndNonExistingKeys() {
+	suite.runWithDefaultClients(func(client api.BaseClient) {
+		key1 := uuid.New().String()
+		key2 := uuid.New().String()
+		key3 := uuid.New().String()
+		oldValue := uuid.New().String()
+		value := uuid.New().String()
+		suite.verifyOK(client.Set(key1, oldValue))
+		keyValueMap := map[string]string{
+			key1: value,
+			key2: value,
+		}
+		suite.verifyOK(client.MSet(keyValueMap))
+		keys := []string{key1, key2, key3}
+		values := []string{value, value, ""}
```

Comment:
```suggestion
		values := []string{value, value, nil}
```
`mset` should return null for missing key

Reply:
In GO, for string dataType nil is not supported. Hence, we are returning empty string in case of all the commands if the key is not present.

Reply:
This need to be reflected in docs and/or in examples

Reply:
It is covered in the description of the commands.